packages/suite/src/views/wallet/transactions/components/TransactionList/index.tsx

```diff
@@ -81,6 +91,127 @@ const TransactionList = ({
         slicedTransactions,
     ]);
 
+    const { translationString } = useTranslation();
+    const { addNotification } = useActions({
+        addNotification: addToast,
+    });
+    const [isExportRunning, setIsExportRunning] = useState(false);
+    const { symbol, descriptor } = account;
+    const runExport = useCallback(
+        async type => {
+            // Don't run again if already running
+            if (isExportRunning) {
+                return;
```

Comment:
Maybe an error toast could happen here but not sure

Reply:
This shouldn't never happen and in case it does, I don't think we should show anything to the user. The UI should display a loading spinner anyway.